.travis.yml

```diff
@@ -5,4 +5,4 @@ matrix:
     - node_js: "0.12"
     - node_js: "io.js"
 before_script:
-  - mysql -e 'create database insidewarehouse_test;'
+  - mysql -e 'create database insidewarehouse_utest;'
```

Comment:
That's what it's called in our vagrant VM - I wanted it to be called the same in Travis, because I use this in asserts.